docs/upgrade/v1-2-1-to-v1-2-2.md

```diff
@@ -13,3 +13,25 @@ title: "Upgrade from v1.2.1 to v1.2.2"
 An **Upgrade** button appears on the **Dashboard** screen whenever a new Harvester version that you can upgrade to becomes available. For more information, see [Start an upgrade](./automatic.md#start-an-upgrade).
 
 For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
+
+## Known issues
+
+---
+
+### 1. The `harvester-seeder` add-on is not upgraded
+
+When upgrading Harvester clusters from v1.2.1 to v1.2.2, the `harvester-seeder` add-on is not fully upgraded due to the chart version mismatch:
+
+```
+Error: chart "harvester-seeder" matching 0.1.0 not found in harvester-seeder index. (try 'helm repo update'): no chart version found for harvester-seeder-0.1.0
+```
+
+The `harvester-seeder` chart version bundled with Harvester v1.2.2 is 0.1.1. However, the `harvester-seeder` add-on remains to use the 0.1.0 chart. The same situation will happen if you enable the add-on after upgrading to Harvester v1.2.2, you will notice the chart could not be installed successfully with the similar messages as above.
+
+The workaround is to manually update the `harvester-seeder` Addon manifest (if it's already enabled, please disable it before patching):
```

Comment:
```suggestion
If you enable the add-on after upgrading to Harvester v1.2.2, installing the chart will not be possible because of the mentioned mismatch.

The workaround is to manually update the `harvester-seeder` add-on manifest using the following command. You must disable the add-on before patching.
```

Reply:
I moved a sentence to the first paragraph for better flow.